apache/superset
# feat: customize screenshot width for alerts/reports

### SUMMARY


This PR is part 1 of 2. It adds a new input to the Alerts & Reports create/edit modals, as well as the report modal from charts and dashboards. In the new input users can configure a custom width for the screenshots

This PR updates the model, adds the migration to introduce a `custom_width` column (and `custom_height`, though it's not used), and add the affordances to set the custom width in the create/edit flows.

The next PR will read the information from the DB and use it when running the alerts and reports.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


New:

![Screenshot 2023-06-28 at 16-45-58 FCC New Coder Survey 2018](https://github.com/apache/superset/assets/1534870/2693fe7e-f7b9-4983-ba6b-05470bee1095)

![Screenshot 2023-06-28 at 16-47-09 FCC New Coder Survey 2018](https://github.com/apache/superset/assets/1534870/c0ba702a-9704-47b6-85d7-5d9836d962bf)

![Screenshot 2023-06-28 at 16-49-27 FCC New Coder Survey 2018](https://github.com/apache/superset/assets/1534870/b9c7075c-cf0f-4c18-bb4b-02cd52ebbdfd)

### TESTING INSTRUCTIONS


Create a few alerts & reports with custom width, then check the DB:

```
customize_screenshot_width=# SELECT id, custom_width FROM report_schedule;
 id | custom_width
----+--------------
  2 |          760
  3 |          600
  1 |          750
(3 rows)
```

### ADDITIONAL INFORMATION
